utils/R/utils.R

```diff
@@ -618,9 +618,9 @@ load.modelpkg <- function(model){
 ##' @author Istem Fer
 misc.convert <- function(x, u1, u2){
   if(u1 == "umol C m-2 s-1" & u2 == "kg C m-2 s-1"){
-    val <- ud.convert(x, "ug m-2 s-1", "kg ha-1 yr-1") * 12 
+    val <- x / (1e09) * 12 
```

Comment:
It seems that having constants floating around like `1e09` and `12` are here is error prone and difficult to debug. 

The benefit of using udunits is that all of the constants are defined by the Udunits library that is widely used, stable, etc. I am not sure if there is an R library that provides physical constants such as the atomic mass of carbon (or if we want to use it), but in lieu of adding another dependency, we could add a library of physical constants to the PEcAn.utils package.

Here we could use `ud.convert(x, 'ug', 'kg') * atomic_mass_of_carbon`.